src/metakb/transform/civic.py

```diff
@@ -95,6 +104,16 @@ class SourcePrefix(str, Enum):
 class CivicTransform(Transform):
     """A class for transforming CIViC to the common data model."""
 
+    # Cache for normalized concepts. The key is the concept type and value is a
+    # dictionary of mappings from CIViC concept (key) to transformed concept (value)
+    able_to_normalize: ClassVar[Dict[str, Dict]] = {
+        "variations": {},  # will store _VariationCache data
+        "molecular_profiles": {},
+        "diseases": {},
+        "therapeutics": {},
+        "genes": {},
+    }
```

Comment:
I'm not sure about having this as a class var. This will change based on the `harvester_path` and I think the changes should be isolated for each instance

Reply:
I'd moved them out here because some methods in the `Base` class presuppose this field's existence -- I settled for defining an empty dict in `__init__()` and then filling it in when the child classes initialize